src/server/models/parking-lot.ts

```diff
@@ -21,7 +22,7 @@ export const ParkingLotSchema = new Schema<
   },
 );
 
-ParkingLotSchema.methods.toJSON = function (): Thruway.ParkingLot {
+ParkingLotSchema.methods.toPresentationLayer = function (): Thruway.ParkingLot {
```

Comment:
To avoid conflicting with `Domcument`'s interface `toJSON` method, I've implemented `toPresentationLayer`.
The naming changes because I want to make sure this is what the client see when fetching the API

Reply:
Details like passwords or sensitive data must be filtered from these models when returning into this method